apps/ejabberd/src/ejabberd_hooks.erl

```diff
@@ -111,7 +111,7 @@ run(Hook, Host, Args) ->
     case ets:lookup(hooks, {Hook, Host}) of
         [{_, Ls}] ->
             mongoose_metrics:increment_generic_hook_metric(Host, Hook),
-            run1(Ls, Hook, Args);
+            run1(Ls, Hook, mongoose_stanza:new(), Args); % run with empty accumulator
```

Comment:
Since we change all hooks run by `ejabberd_hooks:run` to behave as `ejabberd_hooks:run_fold` can we simply remove the helper function `run1` and use relevant `run_fold` instead?